index.html

```diff
@@ -1431,7 +1432,26 @@ <h5>How to's</h5>
  </section>
  </section>
 </section>  
-  
+
+<section id="cursive" class="subtopic">
+  <h3>Cursive text</h3>
+  <p class="advisement" id="cursive_opacity"><a class="self" href="#cursive_opacity">&#x200B;</a>
+    Overlapping joinings should not be exposed when transparency is applied to the cursive text such as Arabic, Mongolian, and N'Ko.
```

Comment:
I suggest tweaking as follows:
"Overlaps should not be exposed when transparency is applied to the joined letters in cursive text, such as for Arabic, Mongolian, and N'Ko."

Reply:
Updated. Thank you!